torchtitan/utils.py

```diff
@@ -166,15 +166,18 @@ def create_context_parallel_ctx(
     cp_buffers: List[torch.Tensor],
     cp_seq_dims: List[int],
     cp_no_restore_buffers: Set[torch.Tensor],
+    cp_rotate_method: str,
 ):
     try:
         from torch.distributed.tensor.experimental import context_parallel
+        from torch.distributed.tensor.experimental._attention import set_rotate_method
```

Comment:
can be merge these two into one line? either import both, or import `_attention` and use it as prefix when calling the two methods.